test/test_playbooks/server_vars.yml.example

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
The file above (test/test_playbooks/server_vars.yml) should not be in the repository. (see .gitignore)